docs/UsersGuide/source/CDEPS.rst

```diff
@@ -0,0 +1,224 @@
+
+======================
+Building CDEPS in HAFS
+======================
+
+The DAPP keyword in the call to **./compile.sh** in **./sorc/build_forecast.sh** should be set to **-DAPP=HAFS-ALL** to build HAFS with support for data models. The resulting executable can also be used for HAFS runs with active atmosphere and ocean models. 
+
+Unless you are working on wcoss_cray, the DAPP keyword should already be set to HAFS-ALL.
+ 
+The remainder of the build process is the same as described in the :ref:`HAFS installation guide <hafs_build>`.
+
+================================
+Using CDEPS in the HAFS Workflow
+================================
+
+The HAFS workflow can be used to run data model experiments with minimal modifications, which are described below.
+
+Modify the **./rocoto/cronjob_hafs_cdeps.sh** script:
+
+1) Uncomment the definitions of HOMEhafs, dev, and PYTHON3 appropriate for the HPC platform that you are using.
+
+2) Set HOMEhafs to the top-level directory that contains the HAFS scripts and source codes.
+
+3) Near the bottom of the script, review the commands for the three DATM and DOCN experiments, and comment out the commands for any experiments that you do not want to run:
+
+    a) To run the DATM with ERA5, the command is:
+
+    .. code-block:: console
+
+        ${PYTHON3} ./run_hafs.py -t ${dev} 2019082900 00L HISTORY \
+     			config.EXPT=${EXPT} \
+			config.SUBEXPT=${EXPT}_era5 \
+			forecast.output_history=.true. \
+     			../parm/hafs_regional_static.conf \
+     			../parm/hafs_hycom.conf \
+     			../parm/hafs_datm.conf \
+     			../parm/hafs_datm_era5.conf
+
+    b) To run the DOCN with OISST, the command is:
+
+    .. code-block:: console
+
+        ${PYTHON3} ./run_hafs.py -t ${dev} 2019082900 00L HISTORY \
+     			config.EXPT=${EXPT} \
+			config.SUBEXPT=${EXPT}_oisst \
+			forecast.output_history=.true. \
+     			../parm/hafs_regional_static.conf \
+     			../parm/hafs_docn.conf \
+     			../parm/hafs_docn_oisst.conf
+
+    c) To run the DOCN with GHRSST, the command is:
+
+    .. code-block:: console
+
+        ${PYTHON3} ./run_hafs.py -t ${dev} 2019082900 00L HISTORY \
+     			config.EXPT=${EXPT} \
+			config.SUBEXPT=${EXPT}_ghrsst \
+			forecast.output_history=.true. \
+     			../parm/hafs_regional_static.conf \
+     			../parm/hafs_docn.conf \
+     			../parm/hafs_docn_ghrsst.conf
+
+The cycle (e.g., 2019082900) and storm (e.g., 00L) can be modified. The final two files in each command configure the CDEPS data models (see :ref:`appendix-a`). It is probably not necessary to change the configuration unless you want to customize the experiment.
+
+Before submitting the cron script, remember to create the **./parm/system.conf** file and to link the fix files using **./sorc/link_fix.sh**. This process is described by the HAFS Developers Guide. (*come back and link to the correct portion of the developers guide*)
```

Comment:
Do users have to run this script separately? It sounds like `install_hafs.sh` calls the link script when building the app, but it would be good to clarify whether users need to rerun the script when adding new data.

Reply:
@gspetro-NOAA, Typically, ./install_hafs.sh will call and run these steps. However, in situations needed, users can also run these steps manually.